presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -319,69 +324,69 @@ Configuration Properties
+``iceberg.rows-for-metadata-optimization-threshold``    The maximum number of partitions in an Iceberg table to       ``1000``                           Yes                 Yes
                                                         allow optimizing queries of that table using metadata. If
                                                         an Iceberg table has more partitions than this threshold,
                                                         metadata optimization is skipped.
 
                                                         Set to ``0`` to disable metadata optimization.
 
-``iceberg.split-manager-threads``                       Number of threads to use for generating Iceberg splits.       ``Number of available processors``
+``iceberg.split-manager-threads``                       Number of threads to use for generating Iceberg splits.       ``Number of available processors`` Yes                 Yes, only needed on coordinator
 
-``iceberg.metadata-previous-versions-max``              The max number of old metadata files to keep in current       ``100``
+``iceberg.metadata-previous-versions-max``              The max number of old metadata files to keep in current       ``100``                            Yes                 NA
                                                         metadata log.
 
-``iceberg.metadata-delete-after-commit``                Set to ``true`` to delete the oldest metadata files after     ``false``
+``iceberg.metadata-delete-after-commit``                Set to ``true`` to delete the oldest metadata files after     ``false``                          Yes                 NA
                                                         each commit.
 
-``iceberg.metrics-max-inferred-column``                 The maximum number of columns for which metrics               ``100``
+``iceberg.metrics-max-inferred-column``                 The maximum number of columns for which metrics               ``100``                            Yes                 NA
                                                         are collected.
-``iceberg.max-statistics-file-cache-size``              Maximum size in bytes that should be consumed by the          ``256MB``
+``iceberg.max-statistics-file-cache-size``              Maximum size in bytes that should be consumed by the          ``256MB``                          Yes                 Yes, only needed on coordinator
                                                         statistics file cache.
-======================================================= ============================================================= ============
+======================================================= ============================================================= ================================== =================== =========================================
 
 Table Properties
```

Comment:
Can you please also add a section for the C++ support for Table Properties? Just saying it's not suported because write is not implemented yet